### PR TITLE
Update README with instructions for platformio

### LINKED ISSUE #168: .platformio with espidf framework compability
Hello, i noticed that [commit](https://github.com/lvgl/lv_port_esp32/commit/a42c982c912e2f93868ebec21306e7a8c912d2f1) says this project have compability with .platformio build system, but i dont have any luck to compile that. 

For now i have bunch of "undefined reference to `lv_something'" and i don't know how to fix any of those things.

### PR DESCRIPTION
**Pull request by C47D**
I don't use platformio to test your instructions @kisvegabor 😅 

Closes #168

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -243,3 +243,38 @@ There are several development boards based on the ESP32 chip, make sure you assi
...
+#include "lvgl.h"
+
+// #include "driver.h"
+
+#include "demo.h"
+
+int app_main(void)
+{
+    lv_init();
+    // hw_init();
```
Comment (kisvegabor) [inline README.md]:
```suggestion
    /*Initialize your hardware*/
    /* hw_init(); */
```

File: README.md [hunk 2]
```diff
@@ -243,3 +243,38 @@ There are several development boards based on the ESP32 chip, make sure you assi
...
+
+    demo_create();
+
+    // hw_loop();
```
Comment (kisvegabor) [inline README.md]:
```suggestion
    /* Create the UI or start a task for it. 
     * In the end, don't forget to call `lv_task_handler` in a loop */
     /* hw_loop(); */
```

**Review by kisvegabor — COMMENTED** [review]
Look good. I've just suggested adding some descriptive comments.

File: README.md (hunk 2 above)
Comment (C47D) [reply README.md]: Added in 9d6974e

File: README.md (hunk 1 above)
Comment (C47D) [reply README.md]: Added in 9d6974e

**Comment by C47D** [comment]
I think 9d6974e fix both comments, what do you think?

**Comment by kisvegabor** [comment]
Great, thanks!

**Pull request merged**